test/integration/034_redshift_test/test_backup_table_option.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+from test.integration.base import DBTIntegrationTest, use_profile
+
+
+class TestBackupTableOption(DBTIntegrationTest):
+    @property
+    def schema(self):
+        return 'backup_table_option_034'
+
+    @staticmethod
+    def dir(path):
+        return os.path.normpath(path)
+
+    @property
+    def models(self):
+        return self.dir("models_backup")
+
+    @property
+    def project_config(self):
+        return {
+            'config-version': 2,
+            'data-paths': [self.dir('seed')],
+            'seeds': {
+                'quote_columns': False,
+            }
+        }
+
+    def project_config_append(self, new_name, new_element):
+        self._project_config[new_name] = new_element
+
+    def check_backup_param_template(self, test_table_name, backup_is_expected):
+        # Use raw DDL statement to confirm backup is set correctly on new table
+        with open('target/run/test/models_backup/{}.sql'.format(test_table_name), 'r') as ddl_file:
+            ddl_statement = ddl_file.readlines()
+            self.assertEqual('backup no' not in ' '.join(ddl_statement).lower(), backup_is_expected)
+        # Use information schema to confirm backup is set correctly on new table (0 if BACKUP NO is present)
+        if 'view' not in test_table_name:
+            check = "select distinct backup from stv_tbl_perm where name = '{}'".format(test_table_name)
```

Comment:
It looks like [only superusers](https://docs.aws.amazon.com/redshift/latest/dg/r_STV_TBL_PERM.html) can read from `stv_tbl_perm`, which is too bad. I'd rather not give the CircleCI/Azure user access, so let's remove this piece of the integration test (even though it offers compelling confirmation).